fastlane/lanes/build.rb

```diff
@@ -374,16 +378,12 @@ def build_and_upload_prototype_build(scheme:, output_app_name:, appcenter_app_na
       - Pull Request: [##{pr}](https://github.com/#{GITHUB_REPO}/pull/#{pr})\n
     NOTES
 
-    appcenter_upload(
-      api_token: get_required_env('APPCENTER_API_TOKEN'),
-      owner_name: APPCENTER_OWNER_NAME,
-      owner_type: APPCENTER_OWNER_TYPE,
-      app_name: appcenter_app_name,
+    upload_build_to_app_center(
+      name: appcenter_app_name,
       file: lane_context[SharedValues::IPA_OUTPUT_PATH],
       dsym: lane_context[SharedValues::DSYM_OUTPUT_PATH],
       release_notes:,
```

Comment:
Notice this Ruby style is not the one we agreed upon, but RuboCop is not currently configured to warn about it. It instead would warn if `release_notes: release_notes` were used. Whenever we'll update the RuboCop config, this should be autocorrect-able